pipeline/lib/current_bench_json.ml

```diff
@@ -177,8 +187,8 @@ let of_json json =
   | _ -> V1.to_v2 @@ V1.of_json json
 
 let validate t =
-  let open V2 in
   let tbl = Hashtbl.create 16 in
+  let open Latest in
```

Comment:
Where does the `Latest` come from? :)

Reply:
We assign `module Latest = V2` just a few lines above, in this file. We could define a `V3` in future, and `validate` shouldn't need to be changed at that point, with this change.

Reply:
great, thanks! :))